### PR TITLE
added resolve pagerduty incidents docs

### PR DESCRIPTION
**Pull request by stavbernazport**
# Description
Added resolve pagerduty incidents guide to enhance the pagerduty incidents automation guide.

## Added docs pages

Please also include the path for the added docs

- Guide (`docs/guides-and-tutorials/resolve-pagerduty-incident.md`)

## Updated docs pages

Please also include the path for the updated docs

- Guide (icon typo) (`docs/guides-and-tutorials/create-slack-channel-for-reported-incident.md`)

### CODE REVIEW

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+```
+
+<PortApiRegionTip/>
+
+</details>
+
+We also need to create the following secrets in our GitHub repository:
+- `PORT_CLIENT_ID` - Your Port client ID.
+- `PORT_CLIENT_SECRET` - Your Port client secret.
+- `BOT_USER_OAUTH_TOKEN` - The Slack app bot token.
```
Comment (MPTG94) [inline docs/guides-and-tutorials/resolve-pagerduty-incident.md:203]: Maybe add in brackets that they get it from the previous guide

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+## Port Action Configuration
+Let's create the Port Self-service action:
+1. Navigate to the `Self-service` tab in your Port organization.
+2. Press `New action`.
+3. Choose the `Edit JSON` option.
+4. Create the action with the following JSON definition:
+<details>
+
+  <summary><b>Port Action: Resolve Incident (click to expand)</b></summary>
+   :::tip
```
Comment (MPTG94) [inline docs/guides-and-tutorials/resolve-pagerduty-incident.md:214]:
```suggestion
   :::tip action placeholders
```

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+- `<GITHUB-REPO-NAME>` - your GitHub repository name.
+:::
+
+
+```json showLineNumbers
+{
+  "identifier": "pagerduty_resolve_incident",
+  "title": "Resolve Incident",
+  "icon": "pagerduty",
+  "description": "Resolve incident in pagerduty",
```
Comment (MPTG94) [inline docs/guides-and-tutorials/resolve-pagerduty-incident.md:226]:
```suggestion
  "description": "Resolve incident in PagerDuty",
```

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+}
+```
+
+</details>
+
+### Testing the Action
+Let's test our action flow:
+1. Head over to your Port organization.
+2. Navigate to the `Self-service` tab.
+3. Find the new action `ACTION_NAME` and press `Execute`.
```
Comment (MPTG94) [inline docs/guides-and-tutorials/resolve-pagerduty-incident.md:266]: ACTION_NAME is not right

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+4. Fill the Pagerduty Incident you would like to resolve and press `Execute`.
+5. Go to your Slack workspace and make sure the incident channel was archived.
+6. Head over to your Github repository and make sure the issue was closed.
+7. Navigate to your [PagerDuty Incidents](https://app.getport.io/pagerdutyIncidents) entities page. Make sure the incident's status changed to `resolved`.
+
+## Summary 
+While in the previous guide we used Port as the automation orchestrator to manage the incident flow, this action will make it easier to resolve Pagerduty incidents including cleaning up all the additional resources we created for the incident in the automated process.
```
Comment (MPTG94) [inline docs/guides-and-tutorials/resolve-pagerduty-incident.md]:
```suggestion
While in the previous guide we used Port as the automation orchestrator to manage the incident flow, this action will make it easier to resolve PagerDuty incidents including cleaning up all the additional resources we created for the incident in the automated process.
```

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+
+## Next Steps
+- Add an Automation to resolve incidents that were not resolved by our newly created Port action, in order to avoid stale Slack channels and Github issues.
```
Comment (MPTG94) [inline docs/guides-and-tutorials/resolve-pagerduty-incident.md:276]:
```suggestion
- Add an Automation that will enhance the action in this guide, and will automatically trigger the incident resolution workflow, to perform automatic cleanup of incidents.
```

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+
+import PortTooltip from "/src/components/tooltip/tooltip.jsx"
+import PortApiRegionTip from "/docs/generalTemplates/_port_region_parameter_explanation_template.md"
+
+# Resolve Pagerduty incidents
+
+## Overview
+
+Solving incidents efficiently is a crucial part of any production-ready environment. When managing an incident, there are a few base concepts which are important to keep:
+- **Real time notifications** - When an incident has been created, either by an alert or manually, it is important that a push notification will be sent the the relevant owners and stakeholders as soon as possible. This can be in the form of a Slack message, email or any other form of communication.
```
[the remark this answers is not in the document]
Comment (MPTG94) [reply docs/guides-and-tutorials/resolve-pagerduty-incident.md:19]: This also happens in the previous guide, so please fix it there too

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md [hunk 8]
```diff
@@ -0,0 +1,276 @@
...
+
+</details>
+
+### Testing the Action
+Let's test our action flow:
+1. Head over to your Port organization.
+2. Navigate to the `Self-service` tab.
+3. Find the new action `ACTION_NAME` and press `Execute`.
+4. Fill the Pagerduty Incident you would like to resolve and press `Execute`.
+5. Go to your Slack workspace and make sure the incident channel was archived.
```
Comment (MPTG94) [inline docs/guides-and-tutorials/resolve-pagerduty-incident.md:268]: The workflow does not perform any action to archive it from what I can tell, we should probably change this to say - Go to your Slack workspace and make sure that the incident resolution was sent to the channel

**Review by MPTG94 — APPROVED** [review]
Apply fixes and merge

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md [hunk 9]
```diff
@@ -0,0 +1,276 @@
...
+          runId: ${{ env.PORT_RUN_ID }}
+          logMessage: "Resloving Pagerduty incident '${{ env.PD_INCIDENT_ID }}'..."
+
+      - name: Request to Resolve Incident
+        id: resolve_incident
+        uses: fjogeleit/http-request-action@v1
+        with:
+          url: 'https://api.pagerduty.com/incidents'
+          method: 'PUT'
+          customHeaders: '{"Content-Type": "application/json", "Accept": "application/vnd.pagerduty+json;version=2", "Authorization": "Token token=${{ secrets.PAGERDUTY_API_KEY }}", "From": "aknmhlxqagykzpwxxs@cazlq.com"}'
```
[the remark this answers is not in the document]
Comment (stavbernazport) [reply docs/guides-and-tutorials/resolve-pagerduty-incident.md:82]: Will need to match a PagerDuty user that can resolve the issue, I'll figure it out

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md (hunk 8 above)
Comment (stavbernazport) [reply docs/guides-and-tutorials/resolve-pagerduty-incident.md:268]: fixed

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md (hunk 1 above)
Comment (stavbernazport) [reply docs/guides-and-tutorials/resolve-pagerduty-incident.md:203]: Modified, let me know what you think

**Comment by hadar-co** [comment]
@stavbernazport please also add a link to the new guide in the [integration index](https://github.com/port-labs/port-docs/blob/main/docs/integrations-index.md)

**Review by hadar-co — APPROVED** [review]

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md [hunk 10]
```diff
@@ -0,0 +1,284 @@
...
+          logMessage: |
+            Incident '${{ env.PD_INCIDENT_ID }}' resolved successfully 💪🏻
+```
+
+<PortApiRegionTip/>
+
+</details>
+
+
+Make sure you created the following secrets in your GitHub repository (Should have done it in the [automation guide](https://docs.getport.io/guides-and-tutorials/create-slack-channel-for-reported-incident)):
```
Comment (MPTG94) [inline docs/guides-and-tutorials/resolve-pagerduty-incident.md:206]:
```suggestion
Make sure you created the following secrets in your GitHub repository (If you followed the [incident automation guide](https://docs.getport.io/guides-and-tutorials/create-slack-channel-for-reported-incident) then they should all already be in place):
```

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md [hunk 11]
```diff
@@ -0,0 +1,284 @@
...
+### Action backend
+As a backend for the Port Action, create a GitHub Workflow in your repository.
+
+Create a workflow file with the following content:
+
+<details>
+    <summary>`Resolve incident` GitHub workflow YAML</summary>
+
+This workflow is responsible for resolving an incident, notifying the Slack channel and closing the GitHub issue.
+:::tip
```
Comment (MPTG94) [inline docs/guides-and-tutorials/resolve-pagerduty-incident.md:43]:
```suggestion
:::tip Update placeholders
```

File: docs/guides-and-tutorials/resolve-pagerduty-incident.md [hunk 12]
```diff
@@ -0,0 +1,284 @@
...
+   Replace the `<PAEGRDUTY_USER_EMAIL>` placeholder to the user email from the [Prerequisites](#prerequisites) section.
```
Comment (MPTG94) [inline docs/guides-and-tutorials/resolve-pagerduty-incident.md:44]:
```suggestion
   Replace the `<PAEGRDUTY_USER_EMAIL>` placeholder with the user email from the [prerequisites](#prerequisites) section. 
```

**Review by MPTG94 — APPROVED** [review]
Tiny changes

**Pull request merged**